src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+extends _blog.pug
+
+block append subtitle
+  | Year one recap
+
+block content
+  h2 Year one recap
+  +news_header('22 November 2023', 'report_18', '')
+
+  p One year ago Input Labs went live, it is time to review our achievements and our failures.
+
+  h3 Things that went nicely
+  h4 Community
+  ul
+    li Our nice #[em community kept growing], now with 735 discord members. This is by far the thing that makes us more proud, and it is awesome to see people helping each other there. To make it even better, some of the members are experts in their fields (electrical engineers, industrial designers) and contribute with expert advice and knowledge.
```

Comment:
Rephrase. Not "kept growing", but "established" during first year. Current number in separate sentence. Something like
"Input labs managed to establish an excellent community on Discord. The currently 735 members are extremely active, helping each others, as well as us. ..."